duckduckgo/Android
# remove covid promo

Task/Issue URL: https://app.asana.com/0/392891325557410/1171406975871618
Tech Design URL: 
CC: 

**Description**:

Remove the COVID-19 promo

**Steps to test this PR**:
1. Fresh install, go through onboarding, ensure no Covid CTA.
1. Existing install, upgrade, ensure no Covid CTA
1. Existing install, CTA disabled, upgrade, ensure no Covid CTA


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
